File: C_Programming/Study/Q1_Assignment29.c
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name :  ChkAlpha
// Description:     Assignment_29: Q.1 :Accept a character from user & check whether it is alphabet or not(A-Za-z).                                       
// Input : F        
// Output: True
// Input:  & 
// Output: False       
//          
// Date:            15/06/2026
// Author:          Shital Ajit Nikam   
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>
#define TRUE 1
#define FALSE 0

typedef int BOOL;
BOOL ChkAlpha(char ch)
{
    BOOL bFlag = FALSE;

    if((ch >= 'A' && ch <= 'Z') || (ch >= 'a' && ch <= 'z'))
    {
        bFlag = TRUE;
    }
    else
    {
        return bFlag;
    }

    return bFlag;

}

int main()
{
    char cValue = '\0';
    BOOL bRet = FALSE;

    printf("Enter the character\n");
    scanf("%c",&cValue);

    bRet = ChkAlpha(cValue);

    if(bRet == TRUE)
    {
        printf("It is character\n");
    }
    else
    {
        printf("It is not a character\n");
    }

    return 0;
}

File: Python_Programming/Competative/Q1_Assignment29.py
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
#        
# Description: Assignment_29: Q.1: write a program which accept name of file from user & checks whether that file 
#                                      exists in current Directory or not.
# Input  :  Demo.txt       
# Output :  Display whether Demo.txt exists or not.                                    
#                                     
# Date:            19/07/2026
# Author:          Shital Ajit Nikam   
#
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import os

#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
# Function Name: DirectoryScanner: Display whether Demo.txt exists in crrent directory or not.
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////


def DirectoryScanner(FileName): 
    CurrentDirectory = os.getcwd()   
    
    for fname in os.listdir(CurrentDirectory):
        if fname == FileName :
            return True
        else:
            return False               
                

def main():        
        
        Border = "-"*60
        print(Border) 
        
        CurrentDirectory = os.getcwd()        
        
        fname = input("Enter the File name :")
        
        Ret = DirectoryScanner(fname) 
        
        if (Ret == True) :
            print("File is present in current Diretory")
            print("path : ",os.path.join(CurrentDirectory,fname))
        else :
            print("File is not present.")              
            
        print(Border)                    
    
        
if __name__ == "__main__" :
    main()